theia-rust-docker/next.package.json

```diff
@@ -0,0 +1,54 @@
+{
+    "private": true,
+    "theia": {
+        "frontend": {
+            "config": {
+                "applicationName": "Theia Rust Example"
+            }
+        }
+    },
+    "dependencies": {
+        "@theia/core": "next",
+        "@theia/cpp-debug": "0.3.19",
```

Comment:
We have made recent changes to `@theia/cpp-debug`, preparing to release it to NPM for the first time at the end of this month. The version has been stepped to `0.7.0', and so `0.3.19` is not found here, resulting in a failing build. I suggest you replace the specific version by '*' for now; once we release the first time, we'll also be able to release `next` versions for each commit, and so then we can just switch to `next` here as well. 

For now I have made the change locally so I can build and test.

Reply:
ok